### DISCUSSION TITLE
How can we put ssl certificate on eXo platform using zerossl.com

### DISCUSSION THREAD

**Discussion by Kevincioo** [exoplatform/exo-community#18] [Q&A]
version: '2.4'
services:
  web:
    image: exoplatform/nginx:1.21.0
    expose:
      - "80"
    ports:
      - "80:80"
    volumes:
      - ./conf/nginx.conf:/etc/nginx/nginx.conf:ro
    links:
      - exo
  exo:
    image: exoplatform/exo-community:6.3
    environment:
      EXO_PROXY_VHOST: localhost
      EXO_PROXY_PORT: 80
      EXO_DB_HOST: mysql
      EXO_PROXY_SSL: "false"
      EXO_DB_TYPE: mysql
      EXO_DB_NAME: exo
      EXO_DB_USER: exo
      EXO_DB_PASSWORD: my-secret-pw
      EXO_ADDONS_LIST:
      EXO_JVM_LOG_GC_ENABLED: "true"
      EXO_ES_HOST: es
      EXO_MONGO_HOST: mongo
      EXO_MONGO_PORT: 27017
      EXO_MONGO_DB_NAME: chat
    ports:
      - "443:443"
    volumes:
      - exo_data:/srv/exo
      - exo_logs:/var/log/exo
      - $EXO_HOME/webapps/custom-extension-webapp.war:/opt/exo/webapps/custom-extension-webapp.war
    depends_on:
      - mysql

**Comment by rdenarie** [comment]
Hello,
You can follow this documentation : https://docs.exoplatform.org/administration/security.html#enabling-https

Simply update the nginx configuration you should already have
